subworkflows/local/busco_annotation/main.nf

```diff
@@ -46,7 +46,7 @@ workflow BUSCO_ANNOTATION {
         []
     )
     ch_versions                 = ch_versions.mix(BUSCO_BUSCO.out.versions.first())
-    ch_grab                     = GrabFiles(BUSCO_BUSCO.out.busco_dir)
+    ch_grab                     = BUSCO_BUSCO.out.busco_dir.map { meta, dir -> tuple(meta, files(dir.resolve("*/*/full_table.tsv"), checkIfExists: true)) }
```

Comment:
1 - `ch_grab` is passed into ANCESTRAL_GENE as `ch_busco_lep_data.busco_dir` as the full_table.tsv now so the new ch_grab made in ANCESTRAL_GENE is no longer needed (as that was used to get the full_table, again).

Reply:
Sorry, I'm not quite following here. The channel says `busco_dir` but it's not the directory and only the full_table.tsv?